docs/concepts/cluster-administration/device-plugins.md

```diff
@@ -48,9 +48,12 @@ to advertise 2 `vendor-domain/foo`.
 Then, developers can request devices in a
 [Container](/docs/api-reference/{{page.version}}/#container-v1-core)
 specification by using the same process that is used for
-[opaque integer resources](/docs/tasks/configure-pod-container/opaque-integer-resource/).
-In version 1.8, extended resources are supported only as integer resources and must have
-`limit` equal to `request` in the Container specification.
+[opaque integer resources](/docs/tasks/configure-pod-container/opaque-integer-resource/),
```

Comment:
Since OIR is deprecated and has be removed in v1.9, shall we replace it here?

Reply:
+1

Reply:
done.